ospf6d/ospf6_message.c

```diff
@@ -404,7 +404,7 @@ static void ospf6_dbdesc_recv_master(struct ospf6_header *oh,
 
 		if (CHECK_FLAG(dbdesc->bits, OSPF6_DBDESC_MSBIT)) {
 			if (IS_OSPF6_DEBUG_MESSAGE(oh->type, RECV))
-				zlog_debug("Master/Slave bit mismatch");
+				zlog_debug("Master/Secondary bit mismatch");
```

Comment:
I don't think we should change user-visible output that may need to be cross-referenced to the RFC; it'll make it harder for people to debug and understand issues, as well as find the same thing in other implementations.

(Changing it and keeping the old term in braces or something is probably OK.)